integration-tests/gradle-plugin-test/settings.gradle.kts

```diff
@@ -0,0 +1,20 @@
+rootProject.name = "gradle-plugin-test"
+
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+        maven("file://${rootDir.absolutePath}/../../packages/build/m2-buildrepo")
```

Comment:
Shouldn't these use the `testRepository` option as well? At least it feels weird that we can customize the repository path and then hardcode it here?

Reply:
I guess it should, but it is a bit tricky as it is a relative path and this project is at a different level than the other projects using it. I would deem it for another PR to fix this up, but I can add a `TODO`.

Reply:
TODO is fine